docs/_pages/releases.md

```diff
@@ -5,6 +5,72 @@ layout: releases
+    This implementation uses Datasource for writing to storage with support for key generators to operate on Row 
+    (rather than HoodieRecords as per previous "bulk_insert") is added.
+  - # TODO Add more about bulk insert modes. 
+  - # TODO Add more on bootstrap.             
+  - In previous versions, auto clean runs synchronously after ingestion. Starting 0.6.0, Hudi does cleaning and ingestion in parallel.
+  - Support async compaction for spark streaming writes to hudi table. Previous versions supported only inline compactions.
+  - Implemented rollbacks using marker files instead of relying on commit metadata. Please check the migration guide for more details on this.
+  - A new InlineFileSystem has been added to support embedding any file format as an inline format within a regular file.
+
+#### Query side improvements:
+  - Starting 0.6.0, snapshot queries are feasible via spark datasource. 
+  - In prior versions we only supported HoodieCombineHiveInputFormat for CopyOnWrite tables to ensure that there is a limit on the number of mappers spawned for
+    any query. Hudi now supports Merge on Read tables also using HoodieCombineInputFormat.
+  - Speedup spark read queries by caching metaclient in HoodieROPathFilter. This helps reduce listing related overheads in S3 when filtering files for read-optimized queries. 
+
+#### DeltaStreamer improvements:
+  - HoodieMultiDeltaStreamer: adds support for ingesting multiple kafka streams in a single DeltaStreamer deployment
+  - Added a new tool - InitialCheckPointProvider, to set checkpoints when migrating to DeltaStreamer after an initial load of the table is complete..
+  - Add CSV source support.
+  - Added chained transformer that can add chain multiple transformers.
+
+#### Indexing improvements:
+  - Added a new index `HoodieSimpleIndex` which joins incoming records with base files to index records.
+  - Added ability to configure user defined indexes.
+
+#### Key generation improvements:
+  - Introduced `CustomTimestampBasedKeyGenerator` to support complex keys as record key and custom partition paths.
+  - Support more time units and dat/time formats in `TimestampBasedKeyGenerator`  
+
+#### Developer productivity and monitoring improvements:
+  - Spark DAGs are named to aid better debuggability
+  - Console, JMX, Prometheus and DataDog metric reporters have been added.
+  - Support pluggable metrics reporting by introducing proper abstraction for user defined metrics.
+
+#### CLI related features:
+  - Added support for deleting savepoints via CLI
+  - Added a new command - `export instants`, to export metadata of instants
+
+#### Other features:
+  - Data snapshot exporter is added for usability. Latest version of records as of a certain point in time can be exported as plain parquet files with this tool.
+  - Introduce write committed callback hooks for incremental pipelines to be notified and act on new commits in the timeline.
+
```

Comment:
if you plan to list out any bugs, probably consider 
[HUDI-1068] - HoodieGlobalBloomIndex does not correctly send deletes to older partition when partition path is updated

Reply:
Yeah. I was thinking of highlighting new features and improvements along with any callouts to migration. Including bugs would be a long list of things. So lets leave that. Its capture in release notes anyways.